js/flatbuffers.js

```diff
@@ -816,6 +828,12 @@ flatbuffers.ByteBuffer = function(bytes) {
    * @private
    */
   this.position_ = 0;
+
+  /**
+   * @type {number}
+   * @private
+   */
+  this.size_ = bytes.length;
```

Comment:
what is this needed for? It doesn't appear to be used anywhere.

Reply:
Nothing, forgot to remove it after I thought I needed it to recreate the ByteBuffer. Removed.